azure-pipelines/build.yml

```diff
@@ -0,0 +1,142 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+# This pipeline will be extended to the MicroBuild template
+# The Task 'NuGetCommand@2' has been converted to an output named 'NuGet push to VS feed' in the templateContext section.
+# The Task 'NuGetCommand@2' has been converted to an output named 'NuGet push to NuGet.org' in the templateContext section.
+name: $(Date:yyyyMMdd).$(Rev:r)
+variables:
+- name: BuildParameters.vsVersion
+  value: 17.0
+- name: ArtifactServices.Symbol.AccountName
+  value: microsoftpublicsymbols
+- name: ArtifactServices.Symbol.PAT
+  value: ''
+- name: Build.OfficialRelease
+  value: false
+- name: Codeql.Enabled
+  value: true
+- name: SignType
+  value: real
+- name: TeamName
+  value: VSCX Tools Authoring
+- name: TimestampPackage
+  value: true
+trigger:
+  branches:
+    include:
+    - refs/heads/master
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    sdl:
+      sourceAnalysisPool: <AgentPoolPlaceHolder>
```

Comment:
This placeholder needs to be replaced. Look at Evgeny's PRs.

Reply:
Thanks, just fixed that.